front-end/src/pages/Wait.jsx

```diff
@@ -1,17 +1,47 @@
 import './Wait.css';
 
+import { get } from '../utils/request';
+//import { Link } from "react-router-dom";
+
 import React from 'react';
 
-const fake_users = [{name: "Jen"}, {name: "Tanya"}, {name: "Thomas"}];
-const total = 6;
 
 function Wait() {
+  const [users, setUsers] = React.useState("")
+  const [userTotal, setUserTotal] = React.useState("?")
+  const [groupId, setName] = React.useState("b6034"); //THIS IS TEMPORARY JUST FOR NOW
```

Comment:
```suggestion
  const [groupId, ] = React.useState("b6034"); //THIS IS TEMPORARY JUST FOR NOW
```
This should hopefully fix the failing test on circleCI!